src/index.js

```diff
@@ -266,7 +267,7 @@ const getChartDatasets = async (sprint, start, end) => {
     numWeekdays
   );
   const labels = getChartLabels(
-    isWeekendsIncluded ? numDaysInSprint : numWeekdays
+    isWeekendsIncluded ? numDaysInSprint : (numWeekdays+1)
```

Comment:
Paiseh, trouble you to manual lint can? 

```
npm run lint:fix
```

Will add the linting step to action soon